src/panels/config/helpers/ha-config-helpers.ts

```diff
@@ -359,6 +380,19 @@ export class HaConfigHelpers extends SubscribeMixin(LitElement) {
             .hass=${this.hass}
             narrow
             .items=${[
+              ...(helper.configEntry &&
+              ERROR_STATES.includes(helper.configEntry.state)
+                ? [
+                    {
+                      path: mdiAlertCircle,
+                      label: this.hass.localize(
+                        "ui.panel.config.helpers.picker.error_information"
+                      ),
+                      warning: true,
```

Comment:
```suggestion
```
It's not a destructive action, so let's not put it in red?

Reply:
I only maybe thought red since we use red alert styling for errors in the other integration page. I know it's not quite the same though. 

![image](https://github.com/user-attachments/assets/3e4ca354-fecb-4a74-adf3-4653e1367896)